src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinSpan.cs

```diff
@@ -284,15 +285,11 @@ public void Write(Utf8JsonWriter writer)
                 {
                     if (tag.Value is int intValue)
                     {
-                        writer.WriteNumber(tag.Key, intValue);
-                    }
-                    else if (tag.Value is bool boolVal)
-                    {
-                        writer.WriteBoolean(tag.Key, boolVal);
+                        writer.WriteString(tag.Key, intValue.ToString(CultureInfo.InvariantCulture));
```

Comment:
Are there tests around the serialization?

Reply:
We have some tests that uses `ZipkinSpan`, but not specifically to the serialization itself.